s3/pkg/datastore/driver/driver.go

```diff
@@ -12,9 +12,11 @@ import (
 // define the common driver interface for io.
 
 type StorageDriver interface {
+	BucketCreate(ctx context.Context, input *pb.Bucket) error
 	Put(ctx context.Context, stream io.Reader, object *pb.Object) (dscommon.PutResult, error)
 	Get(ctx context.Context, object *pb.Object, start int64, end int64) (io.ReadCloser, error)
 	Delete(ctx context.Context, object *pb.DeleteObjectInput) error
+	BucketDelete(ctx context.Context, in *pb.Bucket) error
 	// TODO AppendObject
```

Comment:
is this TODO done

Reply:
it's a previous code. which is not done yet